MediaSession.cpp

```diff
@@ -361,7 +365,11 @@ bool MediaKeySession::playreadyGenerateKeyRequest() {
                         DRM_NO_OF(g_rgpdstrRights),
                         _PolicyCallback,
                         nullptr,
+#ifdef NETFLIX
                         &m_oDecryptContext);
+#else
+                        m_oDecryptContext);
+#endif
```

Comment:
Same as above. Elaborate on this please.

Reply:
In PR 3.3, Drm_Reader_Bind() has decrypt context parameter in double pointer. As this is already under PR_3_3, separated with NETFLIX macro. Could you please confirm on which PR version, NETFLIX extension is available?

Reply:
@pawel-stanek-ml I agree the documentation says the first argument is DRM_DECRYPT_CONTEXT *f_pDecryptContext. It is defined here https://github.com/WebPlatformForEmbedded/OCDM-Playready/blob/master/MediaSession.h#L174. So this should just be 'm_oDecryptContext' without the & (and without #ifdef NETFLIX)? Do you understand how the code currently compiles with '&m_oDecryptContext'?

Reply:
@pawel-stanek-ml  @moorthy-bs looking again I suspect &m_oDecryptContext is a bug introduced by @psivasubramanian as part of PR_3_3 support and Metro guys aren't enabling PR_3_3?

Reply:
@petegriffin yes I agree with you. &m_oDecryptContext is a bug.